docs/managed/update-process.md

```diff
@@ -0,0 +1,11 @@
+# Update of managed
+
+Update of `managed` is triggered by sending a [StartUpdate](https://github.com/percona/pmm/blob/6761010b8b30042936c58c022752f6b57581afee/api/serverpb/server.proto#L325) message.
```

Comment:
This endpoint is not just updating `managed`, but the whole PMM Server.

Reply:
Great catch